libbpf-tools/statsnoop.c

```diff
@@ -121,7 +130,8 @@ static void handle_event(void *ctx, int cpu, void *data, __u32 data_sz)
 		ts = (double)(e.ts_ns - start_timestamp) / 1000000000;
 		printf("%-14.9f ", ts);
 	}
-	printf("%-7d %-20s %-4d %-4d %-s\n", e.pid, e.comm, fd, err, e.pathname);
+	printf("%-7d %-20s %-4d %-4d %-10s %-s\n", e.pid, e.comm, fd, err,
```

Comment:
Should we add a new option for syscall name so that the default output is the same as the Python tool ?

Reply:
Thanks, You're right, just add -s argument.